### ISSUE TITLE
Add --print option from istanbul to print the report

### ISSUE THREAD

**Issue by davglass** [gotwarlost/ytestrunner#7]
Can you add the --print option from istanbul to print the coverage report so that it can be displayed on travis?

**Comment by gotwarlost** [comment]
I really don't want to do this, mainly because ytestrunner shouldn't have had that dependency on istanbul in the first place. That was added purely for backwards compatibility and handling some migration paths for internal projects.

The better way to run ytestrunner is to turn coverage off in it and use the `istanbul cover`  or `istanbul test` command instead. That way you can use all istanbul options just like you do for your `vows` tests.

ytestrunner has built-in protections to ensure no double-instrumentation. See:

https://github.com/gotwarlost/ytestrunner/blob/master/lib/coverage/index.js#L199

Bottom line: you can set up your `npm test` script as:

```
istanbul test --print=both ytestrunner
```

Let me know if this works for you.

**Comment by davglass** [comment]
That didn't work for me when I tried it. It's losing some things..

https://github.com/yui/yuidoc/blob/master/package.json#L48

## 

Dav Glass
davglass@gmail.com
blog.davglass.com
- Windows: n. - The most successful computer virus, ever. +
- A computer without a Microsoft operating system is like a dog
  without bricks tied to its head +
- A Microsoft Certified Systems Engineer is to computing what a
  McDonalds Certified Food Specialist is to fine cuisine  +

**Comment by gotwarlost** [comment]
Closing based on pull request that Dav has already merged
